pkg/subctl/cmd/gather/logs.go

```diff
@@ -57,7 +57,7 @@ func gatherPodLogs(podLabelSelector string, info Info) {
 }
 
 func podLogsToFile(pod *corev1.Pod, info Info) error {
-	logRequest := info.ClientSet.CoreV1().Pods(pod.Namespace).GetLogs(pod.Name, &corev1.PodLogOptions{})
+	logRequest := info.ClientSet.CoreV1().Pods(pod.Namespace).GetLogs(pod.Name, &corev1.PodLogOptions{Previous:true})
```

Comment:
We should only use Previous:true if the request with previous:false fails.

Reply:
I guess that we need to try L62 with previous=false, ... if that fails, then we try again with Previous:true

Reply:
Why not get both? We could have scenarios where a pod crashed, caused some failures but then recovered. In such cases we will want old and current logs both.

Reply:
This patch does that.

I have a operator pod that restarted once.
`
submariner-operator       submariner-operator-7979fd59bb-psg98             1/1     Running     1          35h`

And this is reflected in the output
```
⚠ Gathering operator logs 
 ✓ Found 1 pods matching label selector "name=submariner-operator"
 ⚠ Found logs for previous instances of pod submariner-operator-7979fd59bb-psg98
```
and created
cluster2_submariner-operator-7979fd59bb-psg98.log
cluster2_submariner-operator-7979fd59bb-psg98.log.prev